Update screenshots to new UI

llrs-roche opened the pull request:
# Pull Request

<!--- Replace `#nnn` with your issue link for reference. -->

Fixes #1544 

This updates the screenshots of the shiny apps shown on the bootstrap vignette. 
The svg on teal/vignettes/images/teal-app-components-hover.svg used on the "Getting started with teal"  are updated. In Inkscape they look good but on my browser the example doesn't:

<img width="892" height="569" alt="image" src="https://github.com/user-attachments/assets/2fe1c776-15f6-4c50-9a81-8818caa26be7" />

I also deleted some images I couldn't find where were they shown (Not on blueprints or other).

I run `pkgdown::build_article()` to verify that the vignette is updated and looks good.

averissimo commented on vignettes/images/teal-app-components-hover.svg:
The outer border is cut and doesn't fill the whole image and when hovering this artifact is noticeable.. Low priority, but noticeable

llrs-roche commented on vignettes/bootstrap-themes-in-teal.Rmd:
````diff
@@ -14,20 +14,24 @@ knitr::opts_template$set(
 ### `teal.bs_theme` `R` option
 
-```
+```         
````
Sorry, that might have been added by using the visual editor. Will remove it

llrs-roche commented on vignettes/images/teal-app-components.svg:
Fixed, I moved before remembering to lock the layers and didn't check if it was well aligned...

llrs-roche commented:
@averissimo I updated the images and replaced the svg in the vignette by the png (but kept the svg in the folder). The images lose a bit of definition, so let me know if you prefer this or we keep the svg. 

There is a small noticeable effect with the border because one image is 1 pixel longer than the other. But I aligned both borders to their canvas. Let me know if there is anything else to update.

averissimo reviewed (commented):
We got fooled by outdated html comment before the images

Please make the edits on the `draw.io` files under `inst/design` instead of the SVG and then export it to the SVG

llrs-roche commented on vignettes/images/teal-app-components-hover.drawio.png:
I imported the svg but it didn't let me change the colors :(
